### ISSUE TITLE
2019-11-18 Incorrect partial AUCs when extrapolating from same point

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#26]
### Issue

When including multiple partial AUC that extrapolate from the same point. For example for tc015 the partial AUCs 18-505 and 18-550:

- both require extrapolation (final time is 504)
- both extrapolate from the same point (18) 

This results in the same AUC.

```
      SDEID  AUC0_120  AUC80_505 AUC18_505 AUC18_550  AUC0_720  AUC0_505
1 106469843  159513.0   58235.45  144957.0  144957.0  204068.1  204068.1
2 119434169  129250.5   54387.23  122510.6  122510.6  171431.4  171431.4
3  12659123 1158678.4  455358.79 1047471.1 1047471.1 1527374.6 1527374.6
4 133443675  670111.1  224537.42  550947.2  550947.2  846810.7  846810.7
5 147147665 2707675.8 1533806.64 2967803.1 2967803.1 3985682.1 3985682.1
6 162298431  381595.8  110243.82  302533.7  302533.7  464524.6  464524.6
```

This same behaviour occurs when the point shared by the partial AUCs is zero, and does not affect interpolated values (120).

This has been replicated in tc011, tc012 and tc015. This does not occur in tc010. I am yet to test this for tc013, tc014 and tc016, but they may also be affected.

**Comment by jhhughes256** [comment]
### Update

Example for tc012. The assumption that concentrations are zero-ing out at 24 hours are not necessarily negated with this example.

```
      SDEID  AUC0_505  AUC0_720   AUCINFO   AUCINFP  AUCINFOC  AUCINFPC CONCTIME16 CONCTIME17 CONC16 CONC17
1 106469843  204068.1  204068.1  205289.7  205238.5  205289.7  205238.5        336        504   52.0   11.7
2 119434169  171431.4  171431.4  174616.7  174430.6  174616.7  174430.6        336        504   58.6   22.0
3  12659123 1527374.6 1527374.6 1571330.1 1572235.6 1571330.1 1572235.6        336        504  691.0  250.0
4 133443675  846810.7  846810.7  849069.6  849202.5  849069.6  849202.5        336        504  211.0   25.9
5 147147665 3985682.1 3985682.1 4077685.0 4073788.8 4077685.0 4073788.8        336        504 2000.0  670.0
6 162298431  464524.6  464524.6  467398.1  466817.3  467398.1  466817.3        336        504   82.5   28.0
```

In this case the final two concentrations are shown along with AUC 0-infinity. I think there is still some extra AUC that isn't being captured from looking at this, but will try to put something more convincing together tomorrow.

**Comment by jhhughes256** [comment]
### Update

This has been fixed as of 066fa5d8a1.
